js/dom.js

```diff
@@ -349,6 +349,7 @@ function focusAccessibility() {
   // to avoid a full layout recalc we modify the closest one
   focusAccessibility.ELEMENTS = [
     'a',
+    'li',
```

Comment:
Why is it here? This element is non-focusable.

Reply:
Typically, yes, unless you add `tabindex="0"`.

![li](https://user-images.githubusercontent.com/14100003/97659449-6dac3500-1a45-11eb-9aa8-7496fe7c3731.png)

Reply:
In that case I'll add a check for this attribute and remove the element because here only the inherently focusable elements are listed whereas `tabindex` may be added to anything.